homeassistant/components/ambiclimate/config_flow.py

```diff
@@ -114,7 +114,7 @@ async def _get_token_info(self, code: str | None) -> dict[str, Any] | None:
         store = Store[dict[str, Any]](self.hass, STORAGE_VERSION, STORAGE_KEY)
         await store.async_save(token_info)
 
-        return token_info
+        return token_info  # type: ignore[no-any-return]
```

Comment:
Instead of ignoring the error, you can type the variable in line 109.

Reply:
The `type: ignore` will disappear on it's own once the `ambiclimate` dependency adds type information (and includes `py.typed`). I don't see any real benefit in adding the annotation here over the `type: ignore`.